lib/components/Header.dart

```diff
@@ -17,38 +18,34 @@ class CustomSliverHeaderDelegate extends SliverPersistentHeaderDelegate {
       children: [
         Container(
           color: $styles.colors.background,
-          child: Padding(
-            padding: EdgeInsets.symmetric(horizontal: $styles.padding.l),
-            child: Padding(
-              padding: EdgeInsets.only(bottom: $styles.padding.s),
-              child: Column(
-                  mainAxisAlignment: MainAxisAlignment.end,
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                    progress < 1
-                        ? AnimatedOpacity(
-                            duration: const Duration(milliseconds: 200),
-                            opacity: progress < 0.7 ? 1.0 : 0.0,
-                            child: Padding(
-                              padding:
-                                  EdgeInsets.only(bottom: $styles.padding.xs),
-                              child: Text(
-                                monthYear,
-                                style: $styles.text.subheading1
-                                    .copyWith(color: $styles.colors.black),
-                              ),
-                            ))
-                        : const SizedBox.shrink(),
-                    AnimatedDefaultTextStyle(
-                        duration: const Duration(milliseconds: 200),
-                        style: progress < 1
-                            ? $styles.text.heading1
-                                .copyWith(color: $styles.colors.black)
-                            : $styles.text.heading2
-                                .copyWith(color: $styles.colors.black),
-                        child: const Text("In season today")),
-                  ]),
-            ),
+          padding: EdgeInsets.only(
+            left: $styles.padding.l,
+            right: $styles.padding.l,
+            bottom: $styles.padding.s
+          ),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.end,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              if (progress < 1)
```

Comment:
Making running notes of your review. Am I interpreting this correctly? "Ternary when there is a true and false case. If for when there’s just a “show THIS widget when X is true…”"

Reply:
I think this one was less about ternary versus `if`.

The other end of the ternary operator in this original code was to insert a SizedBox.shrink(), which is basically nothing. Inline in a list `[]`, the ternary operator can be hard to read, adds to indentation etc.
Since the options were add this AnimatedOpacity - or - add a nothing widget, just removing the unnecessary widget and making it an if for the first case felt easier to understand.

Reply:
Thanks for clarifying! Makes sense!